frontend/src/app/ui/components/sequencer/sequencer.component.scss

```
@@ -205,3 +206,45 @@ $gap: 4px;
+
+@media (max-width: 768px) {
+  .play-pause-button {
+    display: none;
+  }
+
+  .mobile-play-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    position: absolute;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)